src/main/java/gov/nih/nci/evs/api/model/Synonym.java

```diff
@@ -48,10 +48,6 @@ public class Synonym extends BaseModel implements Comparable<Synonym> {
   private String type;
 
   /** The "code" of the synonym type, so it can be searched by. */
-  // In the future we can use @WriteOnlyProperty
-  // this does not work: @JsonProperty(access = Access.READ_ONLY)
-  @WriteOnlyProperty
-  @Field(type = FieldType.Keyword)
```

Comment:
You don't want to remove the @Field annotation here, just the @WriteOnlyProperty